main.py

```diff
@@ -36,6 +40,7 @@ def do_try(fn, default=None):
         traceback.print_exc()
         return default
 
+
 def run(filename, instance_as_str=True, rule_type=RuleType.ALL):
```

Comment:
I think rather than filtering later on 'Rule disabled' and 'Rule passed'. I'd rather have keyword arguments here, `include_disabled=False, include_passed=False`. Maybe not for now though, what do you think?

Reply:
Yes, that would be cleaner for sure. 
I'll add this in a batch with passed rules cleanup (including variable naming change, better messages, and stuff like that).